app/messages.py

```diff
@@ -21,6 +21,7 @@
     "message": "Mentorship relation does not" " exist."
 }
 USER_NOT_FOUND = {"message": "User not found."}
+USER_NOT_REGISTERED = {"message": "Please register as a user first to get a verification email."}
```

Comment:
Is this message still required??

Reply:
Is this message still present I removed it?

Reply:
ya its there